tests/unit/template_cassdc_test.go

```diff
@@ -124,11 +135,101 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 
 			renderTemplate(options)
 
+			Expect(err).To(BeNil())
 			// Verify both are present
 			// Initcontainer should only have jmx and jolokia
 			Expect(len(cassdc.Spec.PodTemplateSpec.Spec.InitContainers)).To(Equal(3))
 			// Two containers, medusa and cassandra
 			Expect(len(cassdc.Spec.PodTemplateSpec.Spec.Containers)).To(Equal(2))
 		})
+
+		It("setting allowMultipleNodesPerWorker to true", func() {
+			options := &helm.Options{
+				SetValues: map[string]string{
+					"k8ssandra.allowMultipleNodesPerWorker": "true",
```

Comment:
Is this is a duplicate test of the one at line 184, minus the resource verification? If so, can you remove it?